workbench-core/example/app/src/routes/datasetRoutes.ts

```diff
@@ -122,19 +171,26 @@ export function setUpDSRoutes(
   router.post(
     '/datasets/:datasetId/presignedUpload',
     wrapAsync(async (req: Request, res: Response) => {
-      if (req.params.datasetId.match(uuidWithLowercasePrefixRegExp(dataSetPrefix)) === null) {
```

Comment:
same issue. validation of the parameter is removed, but then the parameter is used later.